lib/lexSplit.mll

```diff
@@ -21,7 +21,7 @@ exception Error
 let blank = [' ''\n''\r''\t']
 let non_blank = [^' ''\n''\r''\t']
 let digit = ['0'-'9']
-let printable = ['0'-'9''a'-'z''A'-'Z'':']
+let printable = [^' ''\n''\r''\t'',']
```

Comment:
Are your sure? Not all chars in this set are "printable"...

Reply:
I think this regex is only used to parse variants, via the lexer `string_spaces`, so maybe I should rename it to `not_space_nor_comma`?

Reply:
No need to do that...